src/pages/watch-later/WatchLater.jsx

```diff
@@ -0,0 +1,52 @@
+import { HistoryVideoCard } from "components";
+import { useWatchLater } from "context";
+import React from "react";
+import { Link } from "react-router-dom";
+import "../liked/liked.css";
+
+function WatchLater() {
+  const { watchLater } = useWatchLater();
+  const { data, error, isLoading } = watchLater;
+
+  const authToken = localStorage.getItem("token");
+
+  return (
+    <div className="liked_videos_container">
+      <div className="page_heading">Watch Later Videos</div>
+      {authToken ? (
+        <>
+          {isLoading && <div>Loading your Watch Later videos</div>}
+          {!isLoading && data.length > 0 ? (
+            <div className="liked_videos">
+              {data.map((details) => (
+                <li key={details._id}>
+                  <HistoryVideoCard details={details} type={"fromwatchlater"} />
```

Comment:
those are cards only, i can't play from here, if you can add the play functionality that redirects to video page that would be good

Reply:
Changed in new PR